app/src/main/java/com/example/poste/activities/SharedFolderActivity.java

```diff
@@ -104,15 +104,11 @@ public void onFailure(Call<ResponseBody> call, Throwable t) {
                 e.printStackTrace();
                 Toast.makeText(this, R.string.internal_error, Toast.LENGTH_LONG).show();
             }
-            sendToDashboard();
+            finish();
```

Comment:
sendToDashboard() just started a new Dashboard activity, and didn't finish this one. A finish() is all that is necessary here